ts/src/connections/package.json

```diff
@@ -18,8 +18,8 @@
 		"buffer": "^5.2.1",
 		"debug": "^4.1.1",
 		"vscode-jsonrpc": "^4.0.0",
-		"@microsoft/dev-tunnels-contracts": ">1.1.1",
-		"@microsoft/dev-tunnels-management": ">1.1.1",
+		"@microsoft/dev-tunnels-contracts": ">1.1.6",
+		"@microsoft/dev-tunnels-management": ">1.1.6",
```

Comment:
Unrelated, but do we have to update these versions with every change now? If so it would be better if it was automatic as part of the build/publish process.

Reply:
ya, it is listed as part of the PR comments now when you create a new SDK PR:

> If you updated the TS SDK did you update the dependencies in package.json for connections and management to require a dependency that is > the current published version(Found using npm view @microsoft/dev-tunnels-contracts). This will fix issues where yarn will pull the old version of packages and will cause mismatched dependencies. 

I can create an item for the build/publish task though as that is a good idea